real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/mainFile.java

```diff
@@ -13,40 +13,57 @@
     public static void main(String args[]) {
         config.getInstance();//must be run at least once
                 
-        for (int i = 0; i < args.length; i++) {
-            if (args[i].equalsIgnoreCase("-g")) {
+        for (int i = 0; i < args.length; i++) 
+        {
+            if (args[i].equalsIgnoreCase("-g")) 
+            {
                 config.GUI_ON = false;
-            } else if (args[i].equalsIgnoreCase("+g")) {
+            } 
+            else if (args[i].equalsIgnoreCase("+g")) 
+            {
                 config.GUI_ON = true;
-            } else if (args[i].equalsIgnoreCase("-r")) {
+            } 
+            else if (args[i].equalsIgnoreCase("-r")) 
+            {
                 config.active = false;
-            } else if (args[i].equalsIgnoreCase("+r")) {
+            } 
+            else if (args[i].equalsIgnoreCase("+r")) 
+            {
                 config.active = true;
             }
         }
         
-        if(config.GUI_ON){
+        if(config.GUI_ON)
+        {
             Gui.getInstance();
         }
         
         // Starts the robot
-        if(config.DATA_PLAY_BACK_DEFAULT){
-            try {
+        if(config.DATA_PLAY_BACK_DEFAULT)
+        {
+            try 
+            {
                 bringup_sim();
-            } catch (Exception e) {
+            } 
+            catch (Exception e) 
+            {
                 Gui.close();
                 System.out.println("Unable to bringup simulated robot. Stacktrace omitted because it's really big.");
                 e.printStackTrace();
                 return;
             }
-        } else {
+        } 
+        else 
+        {
             Robot.getInstance();
         }   
     }
     
     //going to start by just connecting to the IMU
-    public static void bringup_sim() throws Exception {
-        if(config.logging){
+    public static void bringup_sim() throws Exception 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1293101028/issues/source?bid=2938665&fileBranchId=2938741#l63)